### PR TITLE
Move dashboard code from PF folder to Container root level

### PR DESCRIPTION
**Pull request by dvail**
## Description

Moves the dashboard code from `/Containers/Dashboard/PatternFly` to `/Containers/Dashboard` now that the legacy code has been removed.

## Checklist
- [ ] Investigated and inspected CI test results

## Testing Performed

Automated tests.

Manual spot testing that new dashboard code continues to load and work correctly.

### CODE REVIEW

**Comment by dvail** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3084** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/3084" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/3084?utm_source=stack-comment).

File: ui/apps/platform/src/Containers/Dashboard/Widgets/ViolationsByPolicyCategory.test.tsx [hunk 1]
```diff
@@ -5,15 +5,15 @@ import '@testing-library/jest-dom/extend-expect';
 
 import renderWithRouter from 'test-utils/renderWithRouter';
 import { mockChartsWithoutAnimation } from 'test-utils/mocks/@patternfly/react-charts';
-import ViolationsByPolicyCategory from 'Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory';
+import ViolationsByPolicyCategory from 'Containers/Dashboard/Widgets/ViolationsByPolicyCategory';
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Dashboard/Widgets/ViolationsByPolicyCategory.test.tsx:8]: In ViolationsByPolicySeverity.test.tsx below, import path is relative and mock is from Containers level.

What do you think about simpler import path here too?

**Review by pedrottimark — APPROVED** [review]
3, 2, 1, and it is done!

File: ui/apps/platform/src/Containers/Dashboard/Widgets/ViolationsByPolicyCategory.test.tsx (hunk 1 above)
Comment (dvail) [reply ui/apps/platform/src/Containers/Dashboard/Widgets/ViolationsByPolicyCategory.test.tsx:8]: Good catch, I'll push the fix.

**Pull request merged**